pkg/registry/registry.go

```diff
@@ -598,10 +598,11 @@ func (r *PlainRegistry) ListArchs(ctx context.Context, imagePullSecret, image st
 		if len(platforms) == 0 {
 			platforms = append(platforms, Platform{Architecture: "amd64", OS: "linux"})
 		}
+		log.DefaultLogger.WithContext(ctx).Printf("For container %s/%s:%s platforms founded: %v\n", registry, image, tag, platforms)
```

Comment:
We should have a better semantic.
1. image is already added to the log context and hence exported in the log line [here](https://github.com/adevinta/noe/pull/129/files#diff-cb61d8f79451b9541de4a8cc0811523a68d15452b2f5971c7618ea5b423cf4ecR571)
2. This sounds more like a debug log line than an info

```suggestion
		log.DefaultLogger.WithContext(ctx).WithField("platforms": platforms).Debug("Found image architecture")
```